backend/amd64/emit.mlp

```diff
@@ -1204,13 +1204,17 @@ let data l =
 
 (* Beginning / end of an assembly file *)
 
-let begin_assembly() =
+let reset_all () =
   X86_proc.reset_asm_code ();
+  reset ();
```

Comment:
Where is this reset coming from? Can you please prefix it with the module name, even if there is an `open` at the top?

Reply:
Good idea; fixed.